x/claim/keeper/hooks.go

```diff
@@ -53,6 +53,14 @@ func (k Keeper) Hooks() Hooks {
 }
 
 // gamm hooks
+func (h Hooks) AfterProposalFailedMinDeposit(ctx sdk.Context, proposalId uint64) {
+
+}
+
+func (h Hooks) AfterProposalVotingPeriodEnded(ctx sdk.Context, proposalId uint64) {
+
+}
+
```

Comment:
Not super sure why these are added, should they be deleted?

Reply:
Yeah, if they're not needed in the claims module, we should remove.

Reply:
Ah its for interface matching Claims `Hooks` as GovHooks. Probably not needed, but we're already adding extra hooks for gamm as it is. This can be resolved later.